DuckDuckGo/Application/AppDelegate.swift

```diff
@@ -98,10 +98,13 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
     var privacyDashboardWindow: NSWindow?
 
     private(set) lazy var historyViewActionsManager: HistoryViewActionsManager = HistoryViewActionsManager()
-    private(set) lazy var newTabPageActionsManager: NewTabPageActionsManager = NewTabPageActionsManager(
+    private(set) lazy var newTabPageCoordinator: NewTabPageCoordinator = NewTabPageCoordinator(
```

Comment:
NewTabPageCoordinator is a new component that wraps the actions manager and handles sending a pixel on "did appear" event.